### PR TITLE
30_differential_abundance.qmd

### PR DESCRIPTION
**Pull request by jkc9886**
New PR for editing Chapter 10, changes so far :

1. grammar/ language changes.
2. formatting until line 90

### CODE REVIEW

**Comment by jkc9886** [comment]
Does this PR reflect the changes you suggested this morning?

**Review by antagomir — APPROVED** [review]
seems good!

**Comment by jkc9886** [comment]
Could you merge this pull request? I am not authorized to do so.

**Comment by antagomir** [comment]
@TuomasBorman is the merging lord

File: inst/pages/30_differential_abundance.qmd [hunk 1]
```diff
@@ -112,7 +129,7 @@ threshold of 10%.
 
 ```{r prep-daa-data}
 # Agglomerate by genus and subset by prevalence
-tse <- subsetByPrevalent(tse,
+tse <- subsetByPrevalentFeatures(tse,
```
Comment (TuomasBorman) [inline inst/pages/30_differential_abundance.qmd]: subsetByPrevalent is the new name of the function. Update your mia directly from Github

**Review by TuomasBorman — APPROVED** [review]

**Pull request merged**